test/0.10/test.create-topics.kafka.sh

```diff
@@ -22,7 +22,7 @@ testCreateTopics() {
 		echo "Validating topic '$TOPIC'"
 
 		EXISTS=$(/opt/kafka/bin/kafka-topics.sh --zookeeper "$KAFKA_ZOOKEEPER_CONNECT" --list --topic "$TOPIC")
-		POLICY=$(/opt/kafka/bin/kafka-configs.sh --zookeeper "$KAFKA_ZOOKEEPER_CONNECT" --entity-type topics --entity-name "$TOPIC" --describe | awk -F'cleanup.policy=' '{print $2}')
+		POLICY=$(/opt/kafka/bin/kafka-configs.sh --zookeeper "$KAFKA_ZOOKEEPER_CONNECT" --entity-type topics --entity-name "$TOPIC" --describe | grep 'Configs for topic' | awk -F'cleanup.policy=' '{print $2}')
```

Comment:
Could you explain why we need to change the test? Is this new behaviour in 2.5.0 ?

Reply:
The output slightly changed with a deprecation warning. I put this in the commit message:

Needed to update the output check for one test since a new deprecation
warning was breaking the awk.

bash-4.4# /opt/kafka/bin/kafka-configs.sh --zookeeper zookeeper:2181 --entity-type topics --entity-name "compact-1587412989" --describe
Warning: --zookeeper is deprecated and will be removed in a future version of Kafka.
Use --bootstrap-server instead to specify a broker to connect to.
Configs for topic 'compact-1587412989' are cleanup.policy=compact,compression.type=snappy

A simple grep for "Configs for topic" fixed the issue.